facebook/__init__.py

```diff
@@ -311,6 +311,26 @@ def get_app_access_token(self, app_id, app_secret, offline=False):
             return self.request("{0}/oauth/access_token".format(self.version),
                                 args=args)["access_token"]
 
+    def get_code_from_token(self, app_id, app_secret, redirect_uri):
+        """
+        Gets a code to be exchanged for a long lived token.
+        Uses a pre-existing server side long lived token to return a code
+        that can be redeemed for a client side long lived token.
+        """
```

Comment:
This docstring suffers from the same issues as the description in the documentation.